### DISCUSSION TITLE
How to get vertical taskbar in Win11 with Open-Shell?

### DISCUSSION THREAD

**Discussion by zilexa** [Open-Shell/Open-Shell-Menu#1862] [Q&A] [answered]
I prefer Open-Shell as it has many options but still looks user-friendly. However, I miss the option to set the taskbar vertically, docked against the left side of the screen ? 
I used to use StartIsBack but paying just to get vertical taskbar back (coming from Win10) makes little sense to me. 
ExplorerPatcher can do it, but the icons (even when set to Large in ExplorerPatcher) are then a bit small (they are bigger with StartIsBack). 
I only use Windows on my work laptop, assigned by employer. Privately I switched to the much, much, much more user friendly Fedora Silverblue, Also much easier to update. It's just for work that I need to go down this rabbithole to restore vertical taskbar.

I searched but cannot find it in Open-Shell?

**Comment by ge0rdi** [comment, marked as answer]
> I searched but cannot find it in Open-Shell?

There is no such functionality in `Open-Shell`.
You will have to use _alternatives_.
